src/main/java/uk/gov/companieshouse/filinghistory/api/model/FilingHistoryOriginalValues.java

```diff
@@ -1,15 +1,77 @@
 package uk.gov.companieshouse.filinghistory.api.model;
 
+import java.time.Instant;
 import java.util.Objects;
 import org.springframework.data.mongodb.core.mapping.Field;
 
 public class FilingHistoryOriginalValues {
 
     @Field("officer_name")
     private String officerName;
+
     @Field("resignation_date")
     private String resignationDate;
 
+    @Field("charge_creation_date")
+    private Instant chargeCreationDate;
+
+    @Field("property_acquired_date")
+    private Instant propertyAcquiredDate;
+
+    @Field("appointment_date")
+    private String appointmentDate;
+
+    @Field("case_start_date")
+    private String caseStartDate;
+
+    @Field("case_end_date")
+    private String caseEndDate;
+
+    @Field("made_up_date")
+    private String madeUpDate;
+
+    @Field("acc_type")
+    private String accType;
+
+    @Field("change_date")
+    private String changeDate;
+
+    @Field("accounting_period")
+    private String accountingPeriod;
+
+    @Field("period_type")
+    private String periodType;
+
+    @Field("extended")
+    private String extended;
```

Comment:
This field does not exist in the CHIPS package nor is it mapped in the consumer. It can be deleted, we should also remove it from the API request spec.